apps/api/src/app/workflows-v2/util/parse-step-variables.ts

```diff
@@ -1,8 +1,8 @@
 import type { JSONSchemaDefinition } from '@novu/shared';
 
 export type LiquidVariable = {
-  type: 'variable';
```

Comment:
I don't think we need this `type: 'variable'`, we know its variable and I haven't found a place where we would check for that.